pkgs/tools/security/networkminer/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, buildDotnetModule
+, fetchurl
+, unzip
+, dos2unix
+, makeWrapper
+, msbuild
+, mono
+}:
+buildDotnetModule rec {
+  pname = "networkminer";
+  version = "2.8";
+
+  src = fetchurl {
+    # Upstream does not provide versioned releases, a mirror has been uploaded
+    # to archive.org
+    url = "https://archive.org/download/networkminer-${builtins.replaceStrings ["."] ["-"] version}/NetworkMiner_${builtins.replaceStrings ["."] ["-"] version}_source.zip";
+    sha256 = "1n2312acq5rq0jizlcfk0crslx3wgcsd836p47nk3pnapzw0cqvv";
+  };
+
+  nativeBuildInputs = [ unzip dos2unix msbuild ];
+
+  patches = [
+    # Store application data in XDG_DATA_DIRS instead of trying to write to nix store
+    ./xdg-dirs.patch
+  ];
+
+  postPatch = ''
+    # Not all files have UTF-8 BOM applied consistently
+    find . -type f -exec dos2unix -m {} \+
+
+    # Embedded base64-encoded app icon in resx fails to parse. Delete it
+    sed -zi 's|<data name="$this.Icon".*</data>||g' NetworkMiner/NamedPipeForm.resx
+    sed -zi 's|<data name="$this.Icon".*</data>||g' NetworkMiner/UpdateCheck.resx
+  '';
+
+  nugetDeps = ./deps.nix;
+
+  buildPhase = ''
+    runHook preBuild
+
+    msbuild /p:Configuration=Release NetworkMiner.sln
+
+    runHook postBuild
+  '';
```

Comment:
We can't use the standard options for this?

Reply:
I could only get this to work with `msbuild` and not with the standard `dotnet build`. Maybe something to do with using mono?